pkg/parser/parser_test.go

```diff
@@ -27,7 +27,7 @@ func (p *testPrinter) Print(_ io.Writer, r *result.FileResults) error {
 }
 
 func testParser() *Parser {
-	return NewParser(rule.DefaultRules, ignore.NewIgnore([]string{}))
+	return NewParser([]*rule.Rule{&rule.TestRule}, ignore.NewIgnore([]string{}))
```

Comment:
Using the default ruleset has the potential to break tests simply by updating a default rule. Using the test rule instead avoids this.